### ISSUE TITLE
Can't logout with today's build

### ISSUE THREAD

**Issue by JonFerraiolo** [maqetta/maqetta#3551]
@billreed63 

With latest builds, logout puts you into an infinite loop with the message "The session has timed out, please log in again. (Auto-redirect in 10 seconds)" But clicking on "log in" or waiting 10 seconds brings you to the back to a page showing that same message.

**Comment by billreed63** [comment]
I think it may be maybe this code in pagedesigner.html

```
                    if (!result || result.userId=='maqettaUser') {
                        Runtime.isLocalInstall=true;
                    }
```

setting the isLocalInstall to true seems to be sending us to the wrong URL on session timeout

when we logout pagedesigner.html gets called which executes:

```
var run = function() {
                    var userInfo;

                    var result = Runtime.getUser();

                    if (result) {
                        Runtime.userName=result.userId;
                        Runtime.userEmail=result.email;
                    } else {
                        //do nothing
                    }
                    if (!result || result.userId=='maqettaUser') {
                        Runtime.isLocalInstall=true;
                    }
```

The `Runtime.getUser();` is a call to the server which gets a 401 not authorized.
`result = null` so the `Runtime.isLocalInstall=true;` is executed. because we had the 401 dojo publishes a io error that we catch and sends us to `Workbench.sesstionTimeOut` where we set redirect URL based on `singleUserMode`  is singleUserMode is  `Runtime.isLocalInstall`

```
var loginHref = '/maqetta/welcome';
    if(Runtime.singleUserMode()) {
        loginHref = '/maqetta/';
    }
```

We should remove all the single user mode code as we do not support single user mode now.

**Comment by jhpedemonte** [comment]
Should be fixed by commits b8b54b2a922518a2c7589a1d6f164f53872361fb (M7) and 9c1f2b526d245ea71c6c110bda25903644bb4de9 (master).  Leaving open to test against builds.
